node/src/chain_spec.rs

```diff
@@ -152,7 +153,7 @@ pub fn development_config() -> Result<ChainSpec, String> {
 pub fn local_config() -> Result<ChainSpec, String> {
     Ok(ChainSpec::builder(
         WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?,
-        None,
+        Default::default(),
```

Comment:
What's the default here ?

Reply:
my bad, this was something I forgot to undo due to some change that was not needed in the end.
Bottom line: No need to do this, I'll undo it.